serializers/src/glTF/2.0/glTFMaterialExporter.ts

```diff
@@ -12,10 +12,9 @@ import { RawTexture } from "babylonjs/Materials/Textures/rawTexture";
 import { Material } from "babylonjs/Materials/material";
 import { StandardMaterial } from "babylonjs/Materials/standardMaterial";
 import { PBRBaseMaterial } from "babylonjs/Materials/PBR/pbrBaseMaterial";
-import { PBRBaseSimpleMaterial } from "babylonjs/Materials/PBR/pbrBaseSimpleMaterial";
 import { PBRMaterial } from "babylonjs/Materials/PBR/pbrMaterial";
 import { PBRMetallicRoughnessMaterial } from "babylonjs/Materials/PBR/pbrMetallicRoughnessMaterial";
-import { PBRSpecularGlossinessMaterial } from "babylonjs/Materials/PBR/pbrSpecularGlossinessMaterial";
+import { IMaterialExtension } from  'babylonjs-gltf2interface';

```

Comment:
nit:
```suggestion
import { IMaterialExtension } from  "babylonjs-gltf2interface";

```

You can change the defaults in vscode:
![image](https://user-images.githubusercontent.com/19158740/117223931-a72e7600-adc3-11eb-8550-e2ccdec89b91.png)

Reply:
Our style is all over the place in this source.

Reply:
fixed